common/generatePPF.js

```diff
@@ -3,49 +3,108 @@ const cp = require('child_process');
 const fs = require('fs');
 const simpleGit = require('simple-git');
 
-module.exports = async (seed, seedName, channel) => {
+function sendReply(patchFilePath,patchFileName,output, channel, interaction,isRace) {
+    if (fs.existsSync(patchFilePath)) {
+        console.log("here");
+        let files = []
+        if(!isRace){
+            files = [{
+                attachment: patchFilePath,
+                name: patchFileName
+            }]
+        }
+        else{
+            channel.then(ch => {
+                ch.send({
+                    content: output,
+                    files: [{
+                        attachment: patchFilePath,
+                        name: patchFileName
+                    }],
+                })
+            }).catch(console.error);
+        }
+        interaction.editReply({ 
+            content: output,
+            files: files
+        ,});
+    }
+}
+
+module.exports = async (seed, seedName, channel, catagory, tournament,interaction, randoMusic, isRace) => {
     console.log(seedName);
     let patchFileName = seedName + ".ppf";
     let randoPath = config.randoPath;
 
-    const git = simpleGit({
```

Comment:
I'm guessing this is done to support unimplemented presets. Understandable, but I would rather maintain this and use a fork for specified presets, both being up to date and public. Additionally something me and Mouse were thinking of is supporting custom preset generation and a library of user submitted presets(with the referee role probably) that would live on the bot side. Thus allowing people to do test races easily.